src/one_dragon_qt/view/devtools/devtools_screen_manage_interface.py

```
@@ -491,63 +558,136 @@ def _on_area_table_cell_changed(self, row: int, column: int) -> None:
+                    if len(template_list) > 1:
+                        area_item.template_sub_dir = template_list[0]
+                        area_item.template_id = template_list[1]
+                    else:
+                        area_item.template_sub_dir = ''
+                        area_item.template_id = template_list[0]
+            elif attr_name == 'pc_rect':
+                area_item.pc_rect = new_value
+                self._image_update.signal.emit()
+            else:
+                setattr(area_item, attr_name, new_value)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)